Dockerfile

```diff
@@ -1,35 +1,87 @@
-FROM r-base:4.0.3
-
-MAINTAINER https://github.com/pacificclimate/chickadee
-LABEL Description="chickadee WPS" Vendor="Birdhouse" Version="0.1.0"
+FROM rocker/r-ver:4.0.3 AS builder
```

Comment:
Why did we move away from `r-base`?

Reply:
I changed it because it seems like `rocker` is a more optimized base image:
```
r-base                  4.0.3     9b4fd0403f8f   12 days ago   1.35GB
rocker/r-ver            4.0.3     d41c9c918189   2 weeks ago   812MB
```